CDCgov/prime-reportstream
# Validation front-end design updates

This PR addresses three outcomes of Florence's design usability research into the Validation service with O&O.

Test Steps:
1. Ensure tests pass
2. Ensure the proper tables are displayed for the proper types of changes requested (errors and warnings)
3. Ensure consistency in banner placement (under the filename)
4. Ensure error messaging links to helpful resources (programmers guide, LIVD table doc)

## Changes
- Combines error and warning tables into single component with enumerated title for differentiation
- No longer renders green banner when warnings are present
- Banner content updates
- Table header and column updates

<img width="784" alt="Screen Shot 2022-10-27 at 4 28 42 PM" src="https://user-images.githubusercontent.com/4022304/198714819-76783192-6019-42a0-9a0f-396cd8481955.png">
<img width="779" alt="Screen Shot 2022-10-28 at 3 14 37 PM" src="https://user-images.githubusercontent.com/4022304/198714829-d2e30bd7-675f-4517-9594-b0dcce204c56.png">
<img width="797" alt="Screen Shot 2022-10-28 at 3 14 17 PM" src="https://user-images.githubusercontent.com/4022304/198714834-318de85b-2bc2-45d6-956a-c5089fd98927.png">


## Checklist

### Testing

### Process

## Linked Issues
- Fixes #7124 
- Fixes #7125 
- Fixes #7126 
- Fixes #7127

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
